types/node/module.d.ts

```diff
@@ -156,10 +154,6 @@ declare module "module" {
              * Export conditions of the relevant `package.json`
              */
             conditions: string[];
-            /**
-             * @deprecated Use `importAttributes` instead
-             */
-            importAssertions: ImportAttributes;
```

Comment:
The docs don't say this has been removed, and I can see the runtime code still using this; is this removal intentional?

Reply:
https://github.com/nodejs/node/pull/52104 dropped support for the syntax and dynamic import's second parameter, but apparently did not remove it fully (e.g. https://github.com/nodejs/node/blob/2d1b4a8cf7ce875f0c458941bb40ba6fbdccd42f/lib/internal/modules/esm/load.js#L112); not sure where to draw the line here but I guess it's probably okay to remove it and see what happens?

Reply:
New major version looks like a good time to remove deprecated  things 😄